src/daft-core/src/array/ops/skew.rs

```diff
@@ -0,0 +1,36 @@
+use arrow2::array::PrimitiveArray;
+use common_error::DaftResult;
+
+use crate::{
+    array::{
+        ops::{DaftSkewAggable, GroupIndices},
+        DataArray,
+    },
+    datatypes::Float64Type,
+    utils::stats,
+};
+
+impl DaftSkewAggable for DataArray<Float64Type> {
+    type Output = DaftResult<Self>;
+
+    fn skew(&self) -> Self::Output {
+        let stats = stats::calculate_stats(self)?;
+        let values = self.into_iter().flatten().copied();
+        let skew = stats::calculate_skew(stats, values);
+        let field = self.field.clone();
+        let data = PrimitiveArray::<f64>::from([skew]).boxed();
+        Self::new(field, data)
+    }
+
+    fn grouped_skew(&self, groups: &GroupIndices) -> Self::Output {
+        let grouped_skew_iter = stats::grouped_stats(self, groups)?.map(|(stats, group)| {
```

Comment:
Potentially dumb question, but why do we need two variants. Seems like the `grouped_` variant can always be derived from the regular.